nixos/tests/all-tests.nix

```diff
@@ -993,6 +993,7 @@ in {
   sanoid = handleTest ./sanoid.nix {};
   saunafs = handleTest ./saunafs.nix {};
   scaphandre = handleTest ./scaphandre.nix {};
```

Comment:
Please use runTest

Reply:
That causes an infinite recursion. Not sure what to do about that.

Reply:
You need to update the test file, look for some inspiration in nixpkgs :)